### PR TITLE
Docs - postgres, truststore, and keystore

### PR DESCRIPTION
**Pull request by jeff-bruemmer**
- Adds a PostgreSQL page that details connection options and links to the SSL mode Postgres docs.
- Minor edits to doc on Oracle connection.
- Update ssh to SSH in UI.
- Adds brief descriptions of truststores and keystores on page discussing certificates.

@jeff303: just need a spot check here. Anything incorrect? Anything missing?

### CODE REVIEW

**Review by frannietrempe — APPROVED** [review]

File: docs/administration-guide/databases/oracle.md [hunk 1]
```diff
@@ -12,25 +12,24 @@ Head to this page, accept the license agreement, and download `ojdbc8.jar`:
...
 
 To configure the client (Metabase) to authenticate the identity of the server (the Oracle server), you may need to
 configure a truststore file that includes the server's root CA, so that the JVM running Metabase trusts its
 certificate chain. Refer to the
-[Oracle documentation](https://docs.oracle.com/javase/8/docs/technotes/tools/unix/keytool.html) on using `keytool` to
-manage key and truststore files, importing certificates, etc. Once you have a truststore file ready, add the
-following JVM options for Metabase:
+[Oracle documentation](https://docs.oracle.com/javase/8/docs/technotes/tools/unix/keytool.html) on using `keytool` to manage key and truststore files, importing certificates, etc.
+
+Once you have a truststore file ready, add the following JVM options for Metabase:
```
Comment (jeff303) [inline docs/administration-guide/databases/oracle.md:32]: We may want to ditch this part, since now the trust store can be configured on a per-database basis (via the connection option) rather than at the JVM level.

File: docs/administration-guide/databases/oracle.md [hunk 2]
```diff
@@ -1,67 +1,52 @@
-## Working with Oracle in Metabase
+# Working with Oracle in Metabase
 
-Starting in v0.20.0, Metabase provides a driver for connecting to Oracle databases. Under the hood, Metabase uses Oracle's JDBC driver; due to licensing restrictions, we can't
-include it as part of Metabase. Luckily, downloading it yourself and making it available to Metabase is straightforward and only takes a few minutes.
+Under the hood, Metabase uses Oracle's JDBC driver, but due to licensing restrictions, we can't include Oracle's driver as part of Metabase, so you'll need to download the driver and make it available to Metabase.
```
Comment (flamber) [inline docs/administration-guide/databases/oracle.md:3]: I actually don't think that's true anymore after Oracle suddenly became less :hankey:heads and started releasing under a different license last year, but I'm not a lawyer and Oracle is known for being in the top 3 of worst licensing company on Earth.

File: docs/administration-guide/databases/oracle.md [hunk 3]
```diff
@@ -1,67 +1,52 @@
...
 
-### Downloading the Oracle JDBC Driver JAR
+## Downloading the Oracle JDBC Driver JAR
 
-You can download the JDBC driver from [Oracle's JDBC driver downloads page](https://www.oracle.com/technetwork/database/application-development/jdbc/downloads/index.html).
-Head to this page, accept the license agreement, and download `ojdbc8.jar`:
+You can download a JDBC driver from [Oracle's JDBC driver downloads page](https://www.oracle.com/technetwork/database/application-development/jdbc/downloads/index.html).
 
-![Oracle JDBC Download](../images/oracle_jdbc_download.png)
+The minimum driver version should be 19c, regardless of which Java version or Oracle version you have.
```
Comment (flamber) [inline docs/administration-guide/databases/oracle.md:9]:
```suggestion
The minimum driver version should be 19c, regardless of which Java version or Oracle Database version you have.
```

File: docs/administration-guide/databases/oracle.md [hunk 4]
```diff
@@ -1,67 +1,52 @@
...
 
-Before downloading this JAR you may need to sign up for a free account with Oracle. We have had success with the latest version at the time of this writing, 19.3 (even with older Oracle 12c databases), but any version _should_ work.
+We recommend using the `ojdbc11.jar` JAR.
```
Comment (flamber) [inline docs/administration-guide/databases/oracle.md:11]: Lets continue to recommend what we're shipping in EE builds. Once we change that, then we change the docs.
```suggestion
We recommend using the `ojdbc8.jar` JAR.
```

File: docs/administration-guide/databases/oracle.md [hunk 5]
```diff
@@ -1,67 +1,52 @@
...
 
-### Adding the Oracle JDBC Driver JAR to the Metabase Plugins Directory
+## Adding the Oracle JDBC Driver JAR to the Metabase plugins directory
 
-Metabase will automatically make the Oracle driver available if it finds the Oracle JDBC driver JAR in the Metabase plugins directory when it starts up.
-All you need to do is create the directory, move the JAR you just downloaded into it, and restart Metabase.
+In your Metabase directory (the directory where you keep and run your metabase.jar), create a directory called `plugins` (if it doesn't already exist.
 
-### Connecting with SSL
+Move the JAR you just downloaded (`ojbc11.jar`) into the plugins directory, and restart Metabase. Metabase will automatically make the Oracle driver available when it starts back up.
```
Comment (flamber) [inline docs/administration-guide/databases/oracle.md:17]:
```suggestion
Move the JAR you just downloaded (`ojdbc8.jar`) into the plugins directory, and restart Metabase. Metabase will automatically make the Oracle driver available when it starts back up. 
```

File: docs/administration-guide/databases/postgresql.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+- prefer
+- require
+- verify-ca
+- verify-full
+
+See the PostgreSQL docs for a table about the different [SSL Modes][ssl-modes], and select the option that works for you.
+
+### SSL root certificate (PEM)
+
+If you set the SSL Mode to either "verify-ca" or "verify-full", you'll need to specify a root certificate (PEM). You have the option of using a **Local file path** or an **Uploaded filed path**. If you're on Metabase Cloud, you'll need to select **Uploaded file path** and upload your certificate.
```
Comment (flamber) [inline docs/administration-guide/databases/postgresql.md:23]: I really don't like the name "Uploaded file path" - I understand what it means after I read the code.
```suggestion
If you set the SSL Mode to either "verify-ca" or "verify-full", you'll need to specify a root certificate (PEM). You have the option of using a **Local file path** or an **Uploaded file path**. If you're on Metabase Cloud, you'll need to select **Uploaded file path** and upload your certificate.
```

File: docs/administration-guide/databases/postgresql.md [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+
+### Authenticate client certificate
+
+#### SSL Client Certificate (PEM)
+
+You have the option of using a **Local file path** or an **Uploaded filed path**. If you're on Metabase Cloud, you'll need to select **Uploaded file path** and upload your certificate.
```
Comment (flamber) [inline docs/administration-guide/databases/postgresql.md:29]:
```suggestion
You have the option of using a **Local file path** or an **Uploaded file path**. If you're on Metabase Cloud, you'll need to select **Uploaded file path** and upload your certificate.
```

File: docs/administration-guide/databases/postgresql.md [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+
+#### SSL Client KEY (PKCS-8/DER or PKCS-12)
+
+Again, you have the option of using a **Local file path** or an **Uploaded filed path**. If you're on Metabase Cloud, you'll need to select **Uploaded file path** and upload your certificate.
```
Comment (flamber) [inline docs/administration-guide/databases/postgresql.md:33]:
```suggestion
Again, you have the option of using a **Local file path** or an **Uploaded file path**. If you're on Metabase Cloud, you'll need to select **Uploaded file path** and upload your certificate.
```

File: docs/administration-guide/secure-database-connections-with-ssl-certificates.md [hunk 9]
```diff
@@ -42,7 +42,7 @@ You'll need to specify the location of the certificate on the server that's runn
 
 For example, when connecting to a PostgreSQL database, you'll need to add two parameters:
 
-- `sslmode`. You can see the full list of options in PostgreSQL's documentation. We recommend you use `verify-full`; it's the most secure, and overhead is minimal.
+- `sslmode`. You can see the full list of options in [PostgreSQL's documentation](https://www.postgresql.org/docs/current/libpq-ssl.html). We recommend you use `verify-full`; it's the most secure, and overhead is minimal.
```
Comment (flamber) [inline docs/administration-guide/secure-database-connections-with-ssl-certificates.md:45]: Wrong library - alternatively this https://jdbc.postgresql.org/documentation/head/connect.html
```suggestion
- `sslmode`. You can see the full list of options in [PostgreSQL's documentation](https://jdbc.postgresql.org/documentation/head/ssl-client.html). We recommend you use `verify-full`; it's the most secure, and overhead is minimal.
```

File: src/metabase/driver/common.clj [hunk 10]
```diff
@@ -186,7 +186,7 @@
   {:name         "let-user-control-scheduling"
    :type         :boolean
    :display-name (deferred-tru "Choose when syncs and scans happen")
-   :description  (deferred-tru "This enables Metabase to scan for additional field values during syncs allowing smarter behavior, like improved auto-binning on your bar charts.")
+   :description  (deferred-tru "By default, Metabase does a lightweight hourly sync and an intensive daily scan of field values. If you have a large database, turn this on to make changes.")
```
Comment (flamber) [inline src/metabase/driver/common.clj:189]: If you change anything in this file, then you cannot backport, so revert these changes and add them in a separate PR only targeting master.

**Review by flamber — APPROVED** [review]

File: docs/administration-guide/01-managing-databases.md [hunk 11]
```diff
@@ -27,7 +27,7 @@ Although connection details differ database to database, in general you'll need
 - [MongoDB (version 3.6 or higher)](databases/mongodb.md) <!-- MongoDB supported version is from https://www.mongodb.com/support-policy -->
 - [MySQL (version 5.7 or higher, as well as MariaDB version 10.2 or higher)](databases/mysql.md)
 - [Oracle](databases/oracle.md)
-- PostgreSQL
+- [PostgreSQL](postgresql.md)
```
Comment (flamber) [inline docs/administration-guide/01-managing-databases.md:30]:
```suggestion
- [PostgreSQL](databases/postgresql.md)
```

File: docs/administration-guide/databases/postgresql.md [hunk 12]
```diff
@@ -0,0 +1,62 @@
...
+
+### Choose when syncs and scans happen
+
+This is a lightweight process that checks for updates to this database’s schema. In most cases, you should be fine leaving this set to sync hourly.
+
+### Periodically refingerprint tables
+
+This enables Metabase to scan for additional field values during syncs allowing smarter behavior, like improved auto-binning on your bar charts.
+
+[ssl-modes]: https://www.postgresql.org/docs/current/libpq-ssl.html
```
Comment (flamber) [inline docs/administration-guide/databases/postgresql.md:61]: Or https://jdbc.postgresql.org/documentation/head/connect.html
```suggestion
[ssl-modes]: https://jdbc.postgresql.org/documentation/head/ssl-client.html
```

**Review by flamber — APPROVED** [review]
Did it reset Oracle changes? Otherwise looks good
